sunshine/src/main/java/io/github/tatools/sunshine/JunitSuite.java

```diff
@@ -1,23 +1,25 @@
 package io.github.tatools.sunshine;
 
+import java.util.List;
+
 /**
  * @author Dmytro Serdiuk (dmytro.serdiuk@gmail.com)
  * @since 06.04.2017
  */
 public class JunitSuite implements Suite<Class<?>[]> {
 
-    private final TestClasses testClasses;
+    private final Suite<List<Test<Class>>> classesAsSuite;
 
-    public JunitSuite(Location location) {
```

Comment:
maybe there is a point of having two constructors `public JunitSuite(Location location) {...}` and `public JunitSuite(Location location, Rule filter) {...}` ?